src/smt/smt_engine.cpp

```diff
@@ -86,6 +86,7 @@
 #include "smt/defined_function.h"
 #include "smt/logic_request.h"
 #include "smt/managed_ostreams.h"
+#include "smt/abduction_solver.h"
```

Comment:
This should be sorted in dictionary order.

Reply:
I've clang-formatted now.